### PR TITLE
[SCB-1829] provide integration test for RPC consumer to read response header

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

**Comment by liubao68** [comment]
Can you add an example(integration tests) for this feature? You can simply add in any demos, they will be run in docker when CI builds. And I'll create an issue to add documents for this feature.

**Comment by liubao68** [comment]
https://issues.apache.org/jira/browse/SCB-1836

**Comment by wujimin** [comment]
> Can you add an example(integration tests) for this feature? You can simply add in any demos, they will be run in docker when CI builds. And I'll create an issue to add documents for this feature.

ok, i will add one IT case in this PR.

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/TestResponseHeader.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+package org.apache.servicecomb.it.testcase;
+
+import static org.junit.Assert.assertEquals;
+
+import org.apache.servicecomb.it.Consumers;
+import org.junit.Test;
+import org.springframework.http.ResponseEntity;
+
+public class TestResponseHeader {
```
Comment (liubao68) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/TestResponseHeader.java]: not add `ITJunitUtils.runWithHighwayAndRest(TestReponseHeader.class)` ?

**Comment by liubao68** [comment]
already have this test case in demo-springmvc. close this PR.

**Pull request closed without merging**